devstack/settings

```diff
@@ -37,11 +37,23 @@ enable_service group-policy
 disable_service tempest
 ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
 # NFP services
-enable_service orchestrator,proxy,proxy_agent,base_configurator
+enable_service orchestrator,proxy,proxy_agent,config_orch
```

Comment:
How can we distinguish between the mode of installation, base mode or advanced mode ?
based on that we need to enable specific services.
for base mode, base_configurator
for advanced mode, config_orch